status/status.go

```diff
@@ -175,7 +175,7 @@ func (s *Status) generateText(ctx context.Context, integrationTestStatusDetail i
 		if err != nil {
 			return "", fmt.Errorf("error while getting all child taskRuns from pipelineRun %s: %w", pipelineRunName, err)
 		}
-		text, err := FormatTestsSummary(taskRuns)
+		text, err := FormatTestsSummary(taskRuns, application.Name, pipelineRunName, namespace)
```

Comment:
If I can make deal with the devil about this change in follow-up story, that would be great.

Reply:
I think this would work in combination with the suggestion from [this comment](https://github.com/redhat-appstudio/integration-service/pull/609#discussion_r1517691389), since then the pipelineRun link would always be present in the header.

Reply:
file that story then